src/core/lib/transport/transport.h

```diff
@@ -198,6 +198,8 @@ struct grpc_transport_stream_op_batch_payload {
         grpc_chttp2_grpc_status_to_http2_error. Send a RST_STREAM with this
         error. */
   struct {
+    // Error contract: the recipient of cancel_error must cause cancel_error
```

Comment:
The term "recipient" is a bit unclear.  Perhaps this should say "the transport"?

Same thing in the two comments below.